File: 프로그래머스/lv2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices)):
        seconds = 0
        if(i == len(prices) - 1):
            answer.append(seconds)
        else:
            for j in range(i + 1, len(prices)):
                if(prices[j] >= prices[i]):
                    seconds += 1
                else:
                    answer.append(seconds + 1)
                    break
            if(len(answer) != i + 1):
                answer.append(seconds)
    return answer

File: 프로그래머스/lv2/42584. 주식가격/주식가격.java
import java.util.*;

class Solution {
    public List<Integer> solution(int[] prices) {
        List<Integer> answer = new ArrayList<>();
        
        for(int i = 0 ; i < prices.length ; i++) {
            int sec = 0;
            int price = prices[i];
            if(i != prices.length - 1) {
                for(int j = i + 1 ;  j < prices.length ; j++) {
                    if(price <= prices[j]) {
                        sec += 1;
                    }
                    else {
                        sec += 1;
                        break;
                    }
                }
                answer.add(sec);
            }
            else {
                answer.add(0);
            }
        }
        
        return answer;
    }
}